ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -748,16 +795,18 @@ export class ObjectDetailComponent implements OnInit {
   }
 
   saveFk() {
-    let updatedFkNames: Record<string, string> = {}
-
-    this.fkArray.value.forEach((fk: IFkTabData, i: number) => {
-      let oldFk = this.fkData[i]
-      if (oldFk.spName !== fk.spName) {
-        updatedFkNames[oldFk.spName] = fk.spName
-      }
+    let spFkArr: IForeignKey[] = []
```

Comment:
getting following error on trying to delete multiple foreign keys: Found duplicate names in input : ,